src/index.js

```diff
@@ -1,12 +1,55 @@
+/* eslint-disable no-trailing-spaces */
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import mainReducer from './reducers/index';
+import { createBook, removeBook } from './actions/index';
+import randomId from './helpers/randomId';
 
-ReactDOM.render(
+const initialState = {
+  books: [{
+    id: randomId(),
+    title: 'The Little Prince',
+    completed: false,
+    category: 'kids',
+  },
+  {
+    id: randomId(),
+    title: 'The Odisey',
+    completed: true,
+    category: 'learning',
+  }],
+  visibilityFilter: 'SHOW_ALL',
+};
+const store = createStore(mainReducer, initialState);
+
+const book1 = {
+  id: randomId(),
+  title: 'Aesop\'s fables',
+  completed: false,
+  category: 'kids',
+};
+createBook(book1, store.dispatch);
+
+const book2 = {
+  id: randomId(),
+  title: 'Lord of the Rings',
+  completed: false,
+  category: 'fiction',
+};
+
+createBook(book2, store.dispatch);
+removeBook(book2, store.dispatch);
```

Comment:
- [ ] It seems some tests were happening here. I think we can make index.js cleaner by adding these books to the initialState object.